src/start/RealmScreen.js

```diff
@@ -68,13 +80,17 @@ class RealmScreen extends PureComponent<Props, State> {
 
     return (
       <Screen title="Welcome" padding scrollView>
-        <Label text="Your server URL" />
+        <Label
+          text="Register or log in to a Zulip organization to get started."
```

Comment:
Also please add it `zulip-mobile/src/i18n/translations/messages_en.json`

Reply:
@jainkuniya OK, I added it to `zulip-mobile/src/i18n/translations/messages_en.json`. Thanks!